gui/functionality/adjust_timeouts.py

```diff
@@ -114,6 +117,8 @@ def item_pressed(self, item_pressed):
         self.current_cluster = item_pressed
         self.refresh_timeouts(item_pressed)
     
-    def load_cluster_names(self):
-        load_environment()
-        self.cluster_names = list(EnvironmentProvider().environment.clusters.keys())
+    def handle_cluster_name_change(self):
```

Comment:
`name_change`? From when we allow to change names of clusters? I suggest something more like `cluster_list_modification` or better, shorter name. @jchyzy any ideas?